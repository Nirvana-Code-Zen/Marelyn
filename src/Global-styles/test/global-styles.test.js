import GlobalStyles from '../global-styles'
import renderer from 'react-test-renderer'

describe('<GlobalSytles/>', () => {
  test('Should be global styles ready', () => {
    const tree = renderer.create(<GlobalStyles />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
