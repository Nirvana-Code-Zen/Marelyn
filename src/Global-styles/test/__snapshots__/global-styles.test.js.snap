// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalSytles/> Should be global styles ready 1`] = `null`;
